docs/extensions/UIElementExtensions.md

```diff
@@ -0,0 +1,55 @@
+---
+title: UIElement Extensions
+author: vgromfeld
+description: UIElementExtensions provides a simple way to extend the UIElement
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, UIElement, extensions
+---
+
+# UIElement Extensions
+
+`UIElementExtensions` provides helpers for `UIElement`.

```

Comment:
Missing API link (I know they don't exist yet, but following the format should make a correct one), you can re-grab this from the [.template.md file](https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/.template.md) in the docs folder.